### ISSUE TITLE
Lorempixel is down

### ISSUE THREAD

**Issue by Zachiah** [faker-js/faker#1571]
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [ ] I am willing to provide a PR.

### Describe the bug

The `faker.image` module generates URLs for a site [lorempixel.com](https://lorempixel.com). This site doesn't exist (presumably it was taken down)

### Environment Info

```shell
I was actually testing this on the json schema faker playground
```

### Which module system do you use?

- [ ] CJS
- [ ] ESM

### Used Package Manager

other

**Comment by ST-DDT** [comment]
We are rewriting the image module in v8 and are removing lorempixel from our provider list:
https://github.com/faker-js/faker/pull/1477

**Comment by Shinigami92** [comment]
Blocked by #1477
We will remove the provider completely in a separate PR
